internal/awsservice/dynamodb.go

```diff
@@ -45,7 +45,7 @@ func AddItemIntoDatabaseIfNotExist(databaseName string, checkingAttribute, check
 				"#second_attribute": checkingAttribute[1],
 			},
 			ExpressionAttributeValues: map[string]types.AttributeValue{
-				":first_attribute":  &types.AttributeValueMemberN{Value: checkingAttributeValue[0]},
+				":first_attribute":  &types.AttributeValueMemberS{Value: checkingAttributeValue[0]},
```

Comment:
Since we are using [UseCase  (a string) and Commit Hash (a string)](https://github.com/aws/amazon-cloudwatch-agent-test/pull/153/files#diff-334efd747575193d3505843e769add29bb2818c73a18ca77c9c4f97105ec3104R100-R105), therefore, we will change from `AttributeValueMemberN` to `AttributeValueMemberS` (number to string)